driver/src/main/scala/com/stratio/sparkta/driver/SparktaClusterJob.scala

```diff
@@ -44,30 +47,35 @@ object SparktaClusterJob extends SparktaSerializer {
     if (checkArgs(args)) {
       Try {
         initSparktaConfig(args(4), args(3))
```

Comment:
One more thing. The correct way may be to use named arguments, but our target must be to remove spark-submit as command call. IMHO :)